features/admin.connections.v1/components/edit/forms/custom-auth-general-details-form.tsx

```diff
@@ -0,0 +1,250 @@
+                                "identifier.description"
+                        ) }
+                        readOnly={ true }
+                    />
+                    <Field.Input
+                        ariaLabel="name"
+                        inputType="resource_name"
+                        name="name"
+                        value={ editingIDP.name }
+                        label={ t("customAuthentication:fields.createWizard.generalSettingsStep.displayName.label") }
+                        placeholder={ t(
+                            "customAuthentication:fields.createWizard.generalSettingsStep.displayName.placeholder"
+                        ) }
+                        required={ true }
+                        maxLength={ 100 }
+                        minLength={ 3 }
+                        data-componentid={ `${_componentId}-form-wizard-display-name` }
+                        width={ 15 }
+                    />
+                    <Field.Input
+                        name="image"
+                        ariaLabel="image"
+                        inputType="url"
+                        label={ "Icon URL" }
+                        required={ false }
+                        placeholder={ t("authenticationProvider:" +
+                                "forms.generalDetails.image." +
+                                "placeholder") }
+                        value={ editingIDP.image }
+                        data-componentid={ `${ _componentId }-idp-image` }
+                        maxLength={
+                                ConnectionUIConstants
+                                    .GENERAL_FORM_CONSTRAINTS.IMAGE_URL_MAX_LENGTH as number
+                        }
+                        minLength={
+                                ConnectionUIConstants
+                                    .GENERAL_FORM_CONSTRAINTS.IMAGE_URL_MIN_LENGTH as number
+                        }
+                        hint="Logo to display in login pages."
+                        readOnly={ isReadOnly }
+                    />
+                    <Field.Textarea
+                        name="description"
+                        ariaLabel="description"
+                        label={ t("authenticationProvider:forms." +
+                            "generalDetails.description.label") }
+                        required={ false }
+                        placeholder={ t("authenticationProvider:forms." +
+                            "generalDetails.description.placeholder") }
+                        value={ editingIDP.description }
+                        data-componentid={ `${ _componentId }-idp-description` }
+                        maxLength={ ConnectionUIConstants.IDP_NAME_LENGTH.max }
+                        minLength={ ConnectionUIConstants.IDP_NAME_LENGTH.min }
+                        hint="A text description of the authenticator."
```

Comment:
Let's use i18n for string literals. Also let's get the UI text reviewed from doc team.

```suggestion
                        hint="A brief description of the authenticator."
```

Reply:
Ack